CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 ### 🐞 Fixed
 
 ### ⬆️ Improved
+- Avoid multiple multiple `ChatClient::connectUser` invocations by sharing the `Call`. [#5439](https://github.com/GetStream/stream-chat-android/pull/5439)
```

Comment:
multiple multiples :)

Reply:
Yes, it is because it avoids even multiple multiple connections :upside_down_face: 
Let me fix this comment ;)